base/workflow/R/run.write.configs.R

```diff
@@ -95,90 +102,96 @@ run.write.configs <- function(settings, write = TRUE, ens.sample.method = "unifo
-                                write.to.db = write)
-    
+    sa.runs <- PEcAn.uncertainty::write.sa.configs(
+      defaults = settings$pfts,
+      quantile.samples = sa.samples,
+      settings = settings,
+      model = model,
+      write.to.db = write
+    )
+
     # Store output in settings and output variables
     runs.samples$sa <- sa.run.ids <- sa.runs$runs
     settings$sensitivity.analysis$ensemble.id <- sa.ensemble.id <- sa.runs$ensemble.id
-    
+
     # Save sensitivity analysis info
     fname <- PEcAn.uncertainty::sensitivity.filename(settings, "sensitivity.samples", "Rdata",
-                                  all.var.yr = TRUE, pft = NULL)
+      all.var.yr = TRUE, pft = NULL
+    )
     save(sa.run.ids, sa.ensemble.id, sa.samples, pft.names, trait.names, file = fname)
-    
-  }  ### End of SA
-  
+  } ### End of SA
+
   ### Write ENSEMBLE
   if ("ensemble" %in% names(settings)) {
-    ens.runs <- PEcAn.uncertainty::write.ensemble.configs(defaults = settings$pfts,
-                                       ensemble.samples = ensemble.samples, 
-                                       settings = settings,
-                                       model = model, 
-                                       write.to.db = write)
-    
+    ens.runs <- PEcAn.uncertainty::write.ensemble.configs(
+      defaults = settings$pfts,
+      ensemble.samples = ensemble.samples,
+      settings = settings,
+      model = model,
+      write.to.db = write
+    )
+
     # Store output in settings and output variables
     runs.samples$ensemble <- ens.run.ids <- ens.runs$runs
     settings$ensemble$ensemble.id <- ens.ensemble.id <- ens.runs$ensemble.id
-    ens.samples <- ensemble.samples  # rename just for consistency
-    
+    ens.samples <- ensemble.samples # rename just for consistency
+
     # Save ensemble analysis info
     fname <- PEcAn.uncertainty::ensemble.filename(settings, "ensemble.samples", "Rdata", all.var.yr = TRUE)
     save(ens.run.ids, ens.ensemble.id, ens.samples, pft.names, trait.names, file = fname)
   } else {
     PEcAn.logger::logger.info("not writing config files for ensemble, settings are NULL")
-  }  ### End of Ensemble
-  
+  } ### End of Ensemble
+
   PEcAn.logger::logger.info("###### Finished writing model run config files #####")
   PEcAn.logger::logger.info("config files samples in ", file.path(settings$outdir, "run"))
-  
+
   ### Save output from SA/Ensemble runs
-  # A lot of this is duplicate with the ensemble/sa specific output above, but kept for backwards compatibility.   
-  save(ensemble.samples, trait.samples, sa.samples, runs.samples, pft.names, trait.names, 
-       file = file.path(settings$outdir, "samples.Rdata"))
+  # A lot of this is duplicate with the ensemble/sa specific output above, but kept for backwards compatibility.
+  save(ensemble.samples, trait.samples, sa.samples, runs.samples, pft.names, trait.names,
+    file = file.path(settings$outdir, "samples.Rdata")
+  )
```

Comment:
revert

Reply:
Ah OK, looks like styler is moving all _named_ arguments on their own line but accepting lines with multiple _unnamed_ args. I don't much like that but it's another place I'd be willing to accept it for the sake of "just let the robot decide"